packages/lab/src/metric/MetricContent.tsx

```diff
@@ -58,7 +63,7 @@ export const MetricContent = forwardRef<HTMLDivElement, MetricContentProps>(
       subtitleId,
     } = useMetricContext();
 
-    const iconSize = size === "large" ? "medium" : size;
+    const iconSize = size === "large" ? 2 : 1;
```

Comment:
iconSize should match Icon's API

Reply:
Do you mean that the `size` prop for `MetricContent` should also be a number as per the new icon API? As the size props is also used to determine which `Figure` component is used as the `ValueComponent`

Reply:
Ah, I believe this is changing in the future. We can re-visit it then